TeslaApp/TeslaApp/SettingsScreen/SettingsView.swift

```diff
@@ -0,0 +1,139 @@
+// SettingsView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Settings view
+struct SettingsView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let teslaText = "Tesla"
+        static let mileageText = "187 km"
+        static let teslaImageName = "teslaMain"
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        backgroundStackView {
+            VStack {
+                headerView
+                carView
+                controlPanelView
+                Spacer()
+                    .frame(height: 40)
+                if settingsViewModel.tagSelected == 1 {}
+                goToAirSystemView
+                Spacer()
+            }
+        }
+        .navigationBarHidden(true)
+    }
+
+    // MARK: - Private Properties
+
+    @StateObject private var settingsViewModel = SettingsViewModel()
+
+    @State private var isAirSystemViewShown = false
+
+    private let imageNames = ["grayLockedIcon", "climateIcon", "grayLighningIcon", "openTrunkIcon"]
+
+    private var headerView: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                Text(Constants.teslaText)
+                    .font(.system(size: 28))
+                    .fontWeight(.bold)
+                    .foregroundColor(.white)
+                Text(Constants.mileageText)
+                    .font(.system(size: 17))
+                    .fontWeight(.semibold)
+                    .opacity(0.4)
+            }
+            Spacer()
+        }
+        .padding(.all, 25)
+    }
+
+    private var goToAirSystemView: some View {
+        NavigationLink(
+            "",
+            destination: AirSystemView(),
+            isActive: $isAirSystemViewShown
+        )
+    }
```

Comment:
```suggestion
    private var goToAirSystemView: some View {
        NavigationLink(
           Constant.emptyString ,
            destination: AirSystemView(),
            isActive: $isAirSystemViewShown
        )
    }
```

В константы можно вынести